README.md

```diff
@@ -19,6 +19,17 @@ versioning will be used just yet. Breaking changes can be released at any time.
 For the original NodeJS-based prototype of Themis Contract, please see the
 `prototype/v1` branch.
 
+## 🙋🏾‍♀️ Get Involved
+
+- Say hi in one of our **chat rooms** :speech_balloon:
+    - [Register][join-zulip] on Zulip, our chat platform
```

Comment:
Do you think we could just link to to the disco-tools channel directly? iiuc, that will route people through the registration process first if needed, and everyone is added to `#introductions` automatically on joining.

Reply:
I tried that and recall that works for login, but it "forgets" the room from the url hash if a newcomer clicks "register" link from that screen. So it might not be quite the benefit you're thinking.

But also, the *link* to the rooms wasn't the main intention, just a bonus. Listing the rooms was to hint at what's there and offer some small introspection to motivate joining (since zulip doesn't allow seeing room list before the click-click-click of signup process).

Anyhow, that was my logic, but I hold it loosely :)

Reply:
That makes sense! I'm convinced. Many thanks for the PR!